### PR TITLE
feat: add markdown to auction info

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Bugfix**

This PR resolves [CX-523]

### Description

- Add support to markdown to auction info
**Before**
Screenshot 2020-10-29 at 18 43 54

**After**
Screenshot 2020-10-29 at 18 32 22

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-523]: https://artsyproduct.atlassian.net/browse/CX-523

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10577,6 +10644,16 @@ react-native-linear-gradient@2.5.6:
   resolved "https://registry.yarnpkg.com/react-native-linear-gradient/-/react-native-linear-gradient-2.5.6.tgz#96215cbc5ec7a01247a20890888aa75b834d44a0"
   integrity sha512-HDwEaXcQIuXXCV70O+bK1rizFong3wj+5Q/jSyifKFLg0VWF95xh8XQgfzXwtq0NggL9vNjPKXa016KuFu+VFg==
 
+react-native-markdown-display@^7.0.0-alpha.2:
```
Comment (brainbicycle) [inline yarn.lock:10647]: Why do we need an alpha version can we stick to a stable version?

**Review by brainbicycle — COMMENTED** [review]
This looks good just wondering if we need to use an alpha version of the dependency. Also I may be making this up but I for some reason thought we had a Markdown component, maybe doesn't support everything here though @ds300

**Comment by ds300** [comment]
yeah we already have a Markdown component. Did you try that one @MounirDhahri ?

**Comment by MounirDhahri** [comment]
Hmm, I wasn't aware of it actually. I will check it out

**Pull request merged**